tests/test_numpy_integration.py

```diff
@@ -42,3 +49,14 @@ def test_store_array_in_doc(self):
             assert i == job.sp.a
             assert (numpy.array([i, i, i]) == job.doc.array).all()
             assert [i] * 3 == job.doc.array
+
+    def test_store_zero_dim_array(self):
+        # Zero-dimensional array
+        value = 1.0
+        job = self.project.open_job(dict(a=numpy.array(value))).init()
```

Comment:
Store a normal Python int (or float if you like 1.0 instead of 1) in the doc instead of a numpy array so that this test only tests the doc and the other tests the statepoint. Also rename this test appropriately (suffix with `_doc`). That way each of these tests is independent and it'll make it just a little easier to pinpoint issues in the future.